AirbrakeSink.cs

```diff
@@ -36,7 +34,23 @@ private Notice BuildNotice(LogEvent logEvent)
             }
             else
             {
-                return _airbrake.BuildNotice(logEvent.RenderMessage());
+                var notice = _airbrake.BuildNotice(logEvent.RenderMessage());
+                if(logEvent is StackTraceLogEvent stackTraceEvent)
+                {
+                    if(notice.Errors.FirstOrDefault() is ErrorEntry error)
+                    {
+                        error.Backtrace = Utils.GetBacktrace(stackTraceEvent.StackTrace);
```

Comment:
is Utils.GetBacktrace missing?

Reply:
the class is contained in the Sharpbrake.Client namespace(Sharpbrake library)